### PR TITLE
chore: Refactor thumbnail

### PR DESCRIPTION
**Pull request by KubesDavid**
# Pull Request Template

## Description

Remove unused code, simplify code and other minor improvements.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/Avatar.vue [hunk 1]
```diff
@@ -16,69 +12,26 @@ export default {
...
-    variant: {
-      type: String,
-      default: 'circle',
-    },
   },
   computed: {
     style() {
-      let style = {
-        width: `${this.size}px`,
-        height: `${this.size}px`,
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/Avatar.vue:52]: @KubesDavid Can you please share the thought process behind removing this code?

File: app/javascript/dashboard/components/widgets/Avatar.vue [hunk 2]
```diff
@@ -16,69 +12,26 @@ export default {
...
         fontSize: `${Math.floor(this.size / 2.5)}px`,
       };
-
-      if (this.backgroundColor) {
-        style = { ...style, backgroundColor: this.backgroundColor };
-      }
-      if (this.color) {
-        style = { ...style, color: this.color };
-      }
-      return style;
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/Avatar.vue:65]: I'm not sure if I understand the change, why was this removed?

File: app/javascript/dashboard/components/widgets/Avatar.vue [hunk 3]
```diff
@@ -16,69 +12,26 @@ export default {
...
-      default: '',
-    },
-    rounded: {
-      type: Boolean,
-      default: true,
-    },
-    variant: {
-      type: String,
-      default: 'circle',
-    },
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/Avatar.vue:46]: The variant prop is used in some places. Any reason why this was removed?

File: app/javascript/dashboard/components/widgets/Avatar.vue (hunk 1 above)
Comment (KubesDavid) [reply app/javascript/dashboard/components/widgets/Avatar.vue:52]: Hey @pranavrajs,
I removed it because size is inherited from the parent element. So there is no need to specify the size again.

File: app/javascript/dashboard/components/widgets/Avatar.vue (hunk 3 above)
Comment (KubesDavid) [reply app/javascript/dashboard/components/widgets/Avatar.vue:46]: Again, roundness is inherited from the parental element.

File: app/javascript/dashboard/components/widgets/Avatar.vue (hunk 2 above)
Comment (KubesDavid) [reply app/javascript/dashboard/components/widgets/Avatar.vue:65]: The color is already specified in the CSS and is not changed anywhere, so it is redundant.

**Review by nithindavid — APPROVED** [review]

**Comment by muhsin-k** [comment]
Tests are broken.

Screenshot 2022-10-28 at 1 55 14 PM
